components/horizontal-video-testimonials/Component.jsx

```diff
@@ -43,13 +43,15 @@ export default function HorizontalVideoTestimonials({
       className={styles.root}
     >
       <Container width="cc" paddingX={false} className={styles.wrapper}>
-        <LinkedHeading
-          title={title}
-          ctaText={ctaText}
-          ctaLink={ctaManualUrl || ctaInternalUrl}
-          isAlwaysLight
-          className={styles.sectionHeading}
-        />
+        {title && (
+          <LinkedHeading
+            title={title}
+            ctaText={ctaText}
+            ctaLink={ctaManualUrl || ctaInternalUrl}
+            isAlwaysLight
+            className={styles.sectionHeading}
+          />
+        )}
```

Comment:
@broleomo I added conditional render to the Horizontal Video Testimonials' LinkedHeading as well. Since we're doing that for the Vertical Video Panels, I figured we should do the same to the horizontal one.